ouroboros-network/test/Test/Ouroboros/Network/BlockFetch.hs

```diff
@@ -8,17 +14,33 @@ import           Test.Tasty (TestTree, testGroup)
 import           Test.Tasty.QuickCheck (testProperty)
 import           Test.ChainGenerators (TestChainFork(..))
 
+import           GHC.Generics (Generic)
+import           Data.Aeson
+import qualified Data.HashMap.Strict as HM
 import           Data.List
 import qualified Data.Set as Set
 import           Data.Set (Set)
 import qualified Data.Map as Map
 import           Data.Map (Map)
+import           Data.Text (pack)
 import           Data.Typeable (Typeable)
 import           Data.Dynamic (fromDynamic)
 import           Control.Monad.IOSim
 import           Control.Tracer (Tracer(Tracer), contramap)
+import           Control.Concurrent (modifyMVar_, threadDelay)
 import           Control.Exception (throw)
 
+import qualified Cardano.BM.Configuration.Model as CM
+import           Cardano.BM.Data.BackendKind (BackendKind (..))
+import           Cardano.BM.Data.LogItem (LogObject)
+import           Cardano.BM.Data.Output (ScribeDefinition (..), ScribeFormat (..), ScribeKind (..),
+                     ScribePrivacy (..))
+import           Cardano.BM.Data.Severity (Severity (..))
+import           Cardano.BM.Data.SubTrace (SubTrace (..))
+import           Cardano.BM.Data.Tracer (ToLogObject (..), ToObject (..))
+import           Cardano.BM.Setup (setupTrace)
+import           Cardano.BM.Trace (appendName)
+
```

Comment:
@365andreas this is just an idea: it would be nice to export the most useful stuff from `Cardano.BM`.  Then importing logging `import qualified Cardano.BM as BM` would have all the most important bits for doing logging in place, without a need to import so many things from different modules.

Reply:
@coot Added a configuration file so the most of the imports were removed.
We will also export most useful functions from one file.